config/buses/kafka/README.md

```diff
@@ -2,6 +2,12 @@
 
 Deployment steps:
 1. Setup [Knative Eventing](../../../DEVELOPMENT.md)
+1. If you want to set the default Knative Bus to Kafka run the following command to edit the Knative Eventing configuration:
```

Comment:
The bus should be created before the configmap is updated, otherwise channels could be created for an invalid bus config. Moving this entry down to `Create Channels that reference the 'kafka' Bus` would mean the bus is ready to support channels (assuming there are no other issues).

Reply:
With the current implementation Channels created for a Flow will all fall under a single ClusterBus. Which reminds me that the new bus will need to be deployed as a ClusterBus.

Reply:
Thanks - I've moved the instructions further down